packages/wallet/src/services/DrepInfoTracker.ts

```diff
@@ -1,28 +1,35 @@
-import { Cardano, DRepInfo, DRepProvider } from '@cardano-sdk/core';
+import { Cardano, DRepInfo } from '@cardano-sdk/core';
 import { Logger } from 'ts-log';
 import { Observable, map, merge, of, withLatestFrom } from 'rxjs';
 import { RetryBackoffConfig } from 'backoff-rxjs';
+import { TrackedDrepProvider } from './ProviderTracker';
 import { coldObservableProvider } from '@cardano-sdk/util-rxjs';
 import { distinctBlock } from './util';
 import { isNotNil } from '@cardano-sdk/util';
 
 type DrepInfoObservableProps = {
-  drepProvider: DRepProvider;
+  drepProvider: TrackedDrepProvider;
   logger: Logger;
   retryBackoffConfig: RetryBackoffConfig;
   refetchTrigger$: Observable<void>;
 };
 
 /** Use DRepProvider to fetch DRepInfos with retry backoff logic */
-export const createDrepInfoColdObservable =
-  ({ drepProvider, retryBackoffConfig, refetchTrigger$, logger }: DrepInfoObservableProps) =>
-  (drepIds: Cardano.DRepID[]) =>
+export const createDrepInfoColdObservable = ({
+  drepProvider,
+  retryBackoffConfig,
+  refetchTrigger$,
+  logger
+}: DrepInfoObservableProps) => {
+  drepProvider.setStatInitialized(drepProvider.stats.getDRepInfo$);
```

Comment:
It would be better to set it as initialized only if this tracker makes 0 requests during initialization (no drep ids). This fix is better than nothing though, happy to merge this first